<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="bt_loginout">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>로그인</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_UserName">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>46</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QStackedWidget" name="st_search_select">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>501</width>
     <height>101</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_5">
    <widget class="QComboBox" name="cb_select_region">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>전체</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>북구</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>동구</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>서구</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>남구</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>광산구</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="line_searchtext">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_search">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>20</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>검색</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_6">
    <widget class="QLineEdit" name="line_d_searchtext">
     <property name="geometry">
      <rect>
       <x>106</x>
       <y>60</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_select_d_code1">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>대분류</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Q-음식</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>F-생활서비스</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>D-소매</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>N-관광/여가/오락</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>P-스포츠</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>R-학문/교육</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>O-숙박</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>L-부동산</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cb_select_d_code2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>중분류</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cb_select_d_region">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>전체</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>북구</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>동구</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>서구</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>남구</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>광산구</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="bt_d_search">
     <property name="geometry">
      <rect>
       <x>406</x>
       <y>60</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>검색</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QCheckBox" name="bt_d_select">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>상세검색</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
